msu/utilities/math.nut

```diff
@@ -0,0 +1,31 @@
+::MSU.Math <- {
+	// returns log2 of _num, floored (as an int)
+	function log2int( _num )
```

Comment:
Didn't we measure c++ log performance to be superior to this? I think there were some issues with rounding, but maybe we should revisit that at some point.

Reply:
I don't believe we measured performance. C++ log returns a float, and `this.Math.floor` also returns float in BB Squirrel.